docs/base-load-engineer-playbook.md

```diff
@@ -56,12 +56,15 @@ another more appropriate party.
 2. SRE processes ticket to release the tag to production
 3. Update [Github Release][github-releases] to current release
 4. Monitor [Sentry Releases][sentry-releases] for new production issues
-5. (On your 1st Tuesday) [Release to stage][Release-to-stage] (tag, Github release notes)
+5. (On your 1st Tuesday) Before releasing to Stage, locally [run e2e tests][run-e2e-tests] against the latest main.
```

Comment:
suggestion (non-blocking): We don't have to run the tests locally. We can use the GitHub Action [here](https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml) to do it.